packages/remark-mdx/test/fixtures/expressions.md

```diff
@@ -14,7 +14,7 @@ Can contain more braces: <x y={{z: true}}/>
 
 ## Attribute expression
 
-<X {/*stuff!*/}/>
+<X {/*stuff!*/...d}/>
```

Comment:
Expressions now need to be valid expressions. JSX (e.g., in Babel) doesn’t allow just a comment here, it must be a valid spread. And we’re now matching that.